### DISCUSSION TITLE
Export/import configuration

### DISCUSSION THREAD

**Discussion by HDInfautre** [templaza/astroid-framework#754] [Q&A] [answered]
hello
I find this framework to be the best of joomla.
However I can't find a way to export the astroid configuration of a site to another website?
I thought it was possible. Otherwise how to do it?
Regards

**Comment by deltapapa01** [comment, marked as answer]
Hi,
It's really simple: you save your current settings as a new Preset Profiles, download it and import the Preset Profiles on your new website. What's really cool is that you can also transfer the settings from the Zero to the Two.
If you use Akeeba Admin Tools, you have to enable the writing of PHP files in the firewall settings, otherwise it won't work. You can find more information about this here in Git.

WBE,
deltapapa

**Reply by HDInfautre** [reply]
Hi
Perfect, I was looking elsewhere for import/export.
Really the best theme :-)
Additional question but I will open another discussion if necessary.
I have a multilingual version. When switching from cassiopea to astroid, I end up with the default multilingual menu but not my menu for the language corresponding to my browser!
Is there a specific setting with Astroid?

I did not understand what does not work with admin tools that I sometimes have! what would not work the import/export?
Wouldn't temporarily disabling admin tools be enough?

**Reply by rinka88** [reply]
> Hi Perfect, I was looking elsewhere for import/export. Really the best theme :-) Additional question but I will open another discussion if necessary. I have a multilingual version. When switching from cassiopea to astroid, I end up with the default multilingual menu but not my menu for the language corresponding to my browser! Is there a specific setting with Astroid?
> 
> I did not understand what does not work with admin tools that I sometimes have! what would not work the import/export? Wouldn't temporarily disabling admin tools be enough?

https://docs.astroidframe.work/header/header/site-menu-method

**Comment by Chacapamac** [comment]
> When switching from cassiopea to astroid, I end up with the default multilingual menu but not my menu for the language corresponding to my browser! 

You can change how Joomla decide to show your multilingual website on arrival by the Joomla Plugins —> **System - Language Filter**

2 Choices:
• Website Define default language
• Visitor Browser Language

![Screen Shot 2024-08-19 at 8 53 56 AM](https://github.com/user-attachments/assets/859fdad1-51c2-484c-a3b1-4d98f4165436)

**Comment by HDInfautre** [comment]
Hi
I decided to open a new thread because I couldn't get it to work in multilanguage mode.
https://github.com/templaza/astroid-framework/discussions/762
